models/comment.rb

```diff
@@ -24,6 +24,7 @@ class Comment < Content
 
   index({author_id: 1, course_id: 1})
   index({_type: 1, comment_thread_id: 1, author_id: 1, updated_at: 1})
+  index({comment_thread_id: 1, author_id: 1, created_at: 1})
```

Comment:
Hmm, I'm not sure what you're asking in the first question.  As far as the second, no.  We would need to run profiling, and capture all queries, and then analyze which indexes got used.  We have no current way to do that.

MY comment above is from literally running that query against production, and looking at the explain.  I suspected this particular query wasn't using the index before we switched off of Compose, but it didn't seem to really be causing much of an issue, and indexes are so slow to mess around with in MongoDB that I didn't bother trying to act on it.